gitegob/TeamWork
# #168837290 response to UI feedback

## What does this PR do?
Respond to the UI feedback

## Description of task to be completed
- add pagination on the user homepage
- add the option to share articles to other platforms
- add a dropdown for more options on the user profile pic
- fixed the app color
- fixed the border radius to a consistent one
- add an admin dashboard
- add a user dashboard

## How should this be manually tested
- clone the repo
- go to UI/index.html and from there, navigate through the app to check for everything mentioned above in Tasks

## What are the relevant pivotal tracker stories?
[#168837290](https://www.pivotaltracker.com/story/show/168837290)
